apps/hash-frontend/src/pages/flows.page/flow-definition/topbar.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+  AngleRightRegularIcon,
+  InfinityLightIcon,
+  PlayIconSolid,
+  Select,
+} from "@hashintel/design-system";
+import type { SxProps, Theme } from "@mui/material";
+import { Box, outlinedInputClasses, Stack, Typography } from "@mui/material";
+import { format } from "date-fns";
+import { useMemo } from "react";
+
+import { Button } from "../../../shared/ui/button";
+import { MenuItem } from "../../../shared/ui/menu-item";
+import { useFlowDefinitionsContext } from "./shared/flow-definitions-context";
+import { useFlowRunsContext } from "./shared/flow-runs-context";
+
+const typographySx: SxProps<Theme> = {
+  color: ({ palette }) => palette.gray[70],
+  fontWeight: 500,
+  lineHeight: 1,
+};
+
+const selectSx: SxProps<Theme> = {
+  background: "transparent",
+  boxShadow: "none",
+  [`& .${outlinedInputClasses.input}`]: {
+    fontSize: 14,
+    padding: "6px 12px",
+    ...typographySx,
+  },
+  "& svg": {
+    fontSize: 14,
+  },
+};
+
+const Divider = () => (
+  <AngleRightRegularIcon
+    sx={{
+      fill: ({ palette }) => palette.gray[50],
+      fontSize: 18,
+      mr: 0.5,
+      ml: 1,
+    }}
+  />
+);
+
+export const Topbar = ({
```

Comment:
Doesn't need to be addressed as part of this PR, if there are still more layout changes to come or changes around how the `Deliverable` section is rendered, but just wanted to make you aware that currently the content of the `Deliverable` section can overflow in a way that pushes the `Run` button off screen. This is a recording from my 13" MB Pro:

https://github.com/hashintel/hash/assets/42802102/7a52ae56-8d7c-4da7-b928-a0a24dab0674

Reply:
Thanks for flagging – I've aded to H-2542 which is to update the deliverables section

Reply:
In addition I think the "deliverable" and "raw output" have actually been mixed up here (i.e. the table of entities is the raw output, and the Google Sheets file is the deliverable).

However, given the changes we plan in H-2542, which combine table/graph/deliverable views under a single "Output" tab, I'm not sure we need to fix this here.

Reply:
It's not easy to tell from the video, but the 'Raw Output' is the table of persisted entities, and the 'Deliverable' is the data in the CSV. So they are the correct way round.

This is a somewhat confusing example because those companies are just made up by the LLM, and there's only one persisted entity, the file, so it looks like the 'file deliverable' is in the table of entities and 'some entities' are in the deliverables table (but they're not, they're just made-up data).